packages/components/test/make-localized-element.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
Suggestion to keep the tests in the same folders as the things they're testing. I've come to prefer that, and I think services team does as well. @hitchcockwill?

Reply:
https://github.com/udacity/fluent-react-utils/issues/21